File: dev-docs/snippets/documents/evict-all.swift
import JsBaoClient

// Evict all locally stored document data. Pass `onlySynced: true` to preserve
// documents that still have unsynced local changes.
func evictAll(client: JsBaoClient) async {
  // #region example
  await client.documents.evictAll(
    options: EvictAllDocumentsOptions(onlySynced: true)
  )
  // #endregion example
}

File: dev-docs/snippets/documents/evict-all.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Evict all locally stored document data. `onlySynced` preserves docs with
// unsynced changes.
export async function evictAll(client: JsBaoClient) {
  // #region example
  await client.documents.evictAll({ onlySynced: true });
  // #endregion example
}
